pkgs/os-specific/linux/vmm_clock/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
   ];
 
   meta = with lib; {
-    broken = kernel.kernelOlder "4.19";
+    broken = kernel.kernelOlder "3.10";
```

Comment:
All of these kernels (older than 3.10) are EOL and not in in nixpkgs anymore, so we could IMO just drop this check.

Reply:
Removed the check, thanks!